frontend/src/components/Board/BoardTitleSection/BoardTitleSection.styled.jsx

```diff
@@ -0,0 +1,117 @@
+import styled from 'styled-components';
+import { breakpoints } from 'rwd/breakpoints';
```

Comment:
Jak zrobisz pulla z maina to tego piku już nie ma. Zamiast tego korzystamy z theme providera. Nie trzeba go importować jak coś. W innych plikach możesz sprawdzić jak się go używa